libs/core-publishable/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '@baf-wallet/crypto';
+export * from '@baf-wallet/chain-info';
```

Comment:
maybe `multi-chain` should also be a default lib? not sure if you agree.

Reply:
Yessir